psydac/fem/tensor.py

```diff
@@ -143,7 +143,10 @@ def ldim( self ):
 
     @property
     def periodic(self):
-        return [V.periodic for V in self.spaces]
+        # [YG, 27.03.2025]: according to the abstract interface of FemSpace,
+        # this property should return a tuple of `ldim` booleans. However, the
+        # spaces in self.spaces seem to be returning a single scalar value.
+        return tuple(V.periodic for V in self.spaces)
```

Comment:
The purpose of this comment is not clear to me: should we change the periodic function in the self.spaces (presumably univariate Spline spaces ?)

Reply:
For me this is hard to say now. Usually the return type is defined by the abstract interface and all subclasses should adhere to it. That said, maybe univariate spaces are "kind of special" in Psydac. I am not even sure that we ever create a `FemField` of a `SplineSpace`...

Reply:
You mean that SplineSpace should maybe not be a FemSpace ?

Reply:
(in that case maybe just add a question as a comment on top of the fem/splines.py module ?)

Reply:
Better to create an issue about this, otherwise we have to run the tests once again...